finance-service-api/src/main/kotlin/com/ajaxproject/financeservice/controller/NatsController.kt

```diff
@@ -0,0 +1,16 @@
+package com.ajaxproject.financeservice.controller
+
+import com.google.protobuf.GeneratedMessageV3
+import com.google.protobuf.Parser
+import io.nats.client.Connection
+
+interface NatsController<ReqT : GeneratedMessageV3, RespT : GeneratedMessageV3> {
+
+    val subject: String
+
+    val connection: Connection
```

Comment:
why do you need connection in this NatsConroller

Reply:
I think connection is unnecessary because I have it in BPP. Thank you for your comments